### ISSUE TITLE
Toolbar background not tinted when using fragments

### ISSUE THREAD

**Issue by jlelse** [afollestad/app-theme-engine#103]
I don't know if this only happens when using fragments, but the toolbar (AppCompat one) isn't tinted

**Comment by jlelse** [comment]
![screenshot_2016-02-25-15-38-24](https://cloud.githubusercontent.com/assets/8822316/13322699/ec58370e-dbd5-11e5-945f-f96d003f0573.png)

That happens always.....

**Comment by jlelse** [comment]
What can I do? Please resolve this issue asap

**Comment by afollestad** [comment]
Sorry, I'm very busy with work right now.

**Comment by jlelse** [comment]
Ok no problem!
I like ATE but I think I nevertheless have to remove it from my project again...

**Comment by PDDStudio** [comment]
@jlelse why? Are you applying your config from within your activity? And is your toolbar in the main activity or is it inside the fragment?

**Comment by jlelse** [comment]
Yes, I'm applying it from inside the activity and the toolbar is in the activity too.

**Comment by PDDStudio** [comment]
@jlelse did you add a default background color for the toolbar? And do you set the color via tag or code?

**Comment by jlelse** [comment]
I think the problem was that I didn't define a primary color in the theme

**Comment by PDDStudio** [comment]
@jlelse okay, so I'm assuming everything is working now?

**Comment by jlelse** [comment]
@PDDStudio For the moment yes, because I removed color changing option
